### DISCUSSION TITLE
could make user navigate to native app in phone and desktop when click to add to calendar

### DISCUSSION THREAD

**Discussion by Aseel-AlAli-ray** [add2cal/add-to-calendar-button#537] [Q&A] [answered]
Once the website is accessed via a phone and the "Add to Calendar" option is selected, users can choose a calendar application such as Google Calendar or Outlook. Upon selection, then can redirect to the respective native app on the phone?

and desktop also

**Comment by Aseel-AlAli-ray** [comment]
@jekuer  can u help please

**Reply by jekuer** [reply, marked as answer]
Unfortunately, it is not possible to determine whether a user has installed a specific from a website (due to OS security reasons).
That's why we cannot safely send somebody to a specific app.
However, apps can register for specific url schemes and watch. In most cases, for example, Google calendar should open when calling a respective url. This is some passive process we cannot control.

There are some cases, where this is not working nicely. With subscription calendars it is kinda tricky. And Outlook usually does not open the app.

**What specific case did you observe?**

What also always helps: If you see another tool showing a different (better) behavior, share it with us - maybe we can learn something new 😊.

**Reply by Aseel-AlAli-ray** [reply]
Thx 
I see Google Calendar works fine from phones and open the app
